tutorials/text/annotations.py

```diff
@@ -1,4 +1,11 @@
 r"""
+.. redirect-from:: /gallery/userdemo/annotate_simple01
+.. redirect-from:: /gallery/userdemo/annotate_simple02
+.. redirect-from:: /gallery/userdemo/annotate_simple03
+.. redirect-from:: /gallery/userdemo/annotate_simple04
+.. redirect-from:: /gallery/userdemo/anchored_box04
+.. redirect-from:: /gallery/userdemo/annotate_simple_coord01
+.. redirect-from:: /gallery/userdemo/annotate_simple_coord03
```

Comment:
The doc build is failing with this (fairly cryptic) warning

`/home/circleci/project/doc/tutorials/text/annotations.rst:31: WARNING: Explicit markup ends without a blank line; unexpected unindent.`

If you look at line 31 of your locally built `doc/tutorials/text/annotations.rst`, it corresponds to this point in the python file.  So I _think_ you just need to add a blank line between the "redirect-from"s and the title.

Reply:
Thanks for the explanation! I've added a blank line.